src/panels/config/integrations/integration-panels/zwave_js/dialog-zwave_js-add-node.ts

```diff
@@ -489,12 +511,22 @@ class DialogZWaveJSAddNode extends LitElement {
                                         ${this._lowSecurity
                                           ? html`<ha-alert
                                               alert-type="warning"
-                                              title="The device was added insecurely"
+                                              title=${this.hass.localize(
+                                                "ui.panel.config.zwave_js.add_node.added_insecurely"
+                                              )}
                                             >
-                                              There was an error during secure
-                                              inclusion. You can try again by
-                                              excluding the device and adding it
-                                              again.
+                                              ${this.hass.localize(
+                                                "ui.panel.config.zwave_js.add_node.added_insecurely_text"
+                                              )}
+                                              ${typeof this
+                                                ._lowSecurityReason !==
+                                              "undefined"
```

Comment:
reverted this one as it caused a TS error. TBH the entire check is redundant because if the value is not 0-8, the translation will just fail and it won't display anything anyway